packages/terra-application-links/src/terra-dev-site/test/application-links/ApplicationTabsWithIcons.test.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import ApplicationTabs from '../../../tabs/ApplicationTabs';
+import testLinksWithIconsConfig from '../common/testLinksWithIconsConfig';
+
+const demoheaderstyles = {
```

Comment:
You'll want to move these styles to a SCSS file and likely want to make the background themable. See this PR for more info: https://github.com/cerner/terra-framework/pull/550